index.html

```diff
@@ -6,7 +6,7 @@
    <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
-    <title>Responsive</title>
+    <title>Responsive website</title>
```

Comment:
Kindly update the title so that it adequately and briefly describes the content of the page 😃 For example:
  - Poor titles:
    - Microverse project | Your portfolio.
    - Portifolio project
  - Better page titles:
    - Mark's portfolio
    - Mark full-stack web developer